airflow/migrations/versions/0038_3_0_0_add_asset_active.py

```diff
@@ -76,13 +72,11 @@ def downgrade():
         batch_op.add_column(
             sa.Column("is_orphaned", sa.Boolean, default=False, nullable=False, server_default="0")
         )
-    with Session(bind=op.get_bind()) as session:
-        session.execute(
-            sa.text(
-                "update dataset set is_orphaned = true "
-                "where exists (select 1 from asset_active "
-                "where dataset.name = asset_active.name and dataset.uri = asset_active.uri)"
-            )
+    op.execute(
+        sa.text(
+            "update dataset set is_orphaned = true "
+            "where exists (select 1 from asset_active "
+            "where dataset.name = asset_active.name and dataset.uri = asset_active.uri)"
```

Comment:
nit: Does `op.execute` need `sa.text`? I think it could directly take the query.

Reply:
Not really needed but it’s part of the recommendation when migrating to sqlalchemy 2. We don’t currently enforce it. 
https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#migration-to-2-0-step-two-turn-on-removedin20warnings

Reply:
```
$ SQLALCHEMY_WARN_20=1 python2 -W always::DeprecationWarning test3.py
test3.py:9: RemovedIn20Warning: The Engine.execute() function/method is considered legacy as of the 1.x series of SQLAlchemy and will be removed in 2.0. All statement execution in SQLAlchemy 2.0 is performed by the Connection.execute() method of Connection, or in the ORM by the Session.execute() method of Session. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9) (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
  engine.execute("CREATE TABLE foo (id integer)")
/home/classic/dev/sqlalchemy/lib/sqlalchemy/engine/base.py:2856: RemovedIn20Warning: Passing a string to Connection.execute() is deprecated and will be removed in version 2.0.  Use the text() construct, or the Connection.exec_driver_sql() method to invoke a driver-level SQL string. (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
  return connection.execute(statement, *multiparams, **params)
```

Reply:
Alembic should do that for us when using `op.execute` (`~alembic.operations.Operations.execute`). I can understand we need it for `conn.execute(` (where `conn` is `~sqlalchemy.engine.Connection`).

Check Alembic docs here: https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.execute.params.sqltext

<img width="771" alt="image" src="https://github.com/user-attachments/assets/4cfa9483-d109-4c81-a3fa-657098019d44" />

Reply:
Oh, nice. Thanks!

Reply:
Resolved